tools/build.py

```diff
@@ -16,21 +16,21 @@
 
 import argparse
 import multiprocessing
+from os import makedirs, path, uname
```

Comment:
Personally I like the `import os` variant better (just like in other python files). Maybe it would make sense to use it the same way in all python scripts.